eos_data_distribution/avahi_monitor.py

```diff
@@ -188,6 +192,12 @@ def remove_nexthop(self, faceUri):
         return check_call(["nfdc", "remove-nexthop",
                            str(SUBSCRIPTIONS_SOMA), faceUri])
 
+    def start_edge(self):
+        return check_call(["systemctl", "start", "edd-soma-subscriptions-producer"])
```

Comment:
crash: if you `stop` systemd will not try to restart. if it crashes, it's systemd's job to restart it.
`systemctl start` should not fail or something is wronger... you don't implement that in scripts that run it, you don't need to implement that in avahi.

about another service in NDN we'll solve it if/when we get there. it's the same problem anyway, you'd need to add that to ssp acording to your design, and i would like it even less.